couch/check_couch2.py

```diff
@@ -0,0 +1,89 @@
+# (C) Datadog, Inc. 2010-2016
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+# stdlib
+from urlparse import urljoin
+
+# 3rd party
+import requests
+
+# project
+from checks import AgentCheck
+
+class CouchDB2:
```

Comment:
Merge this into one class. We want the basic api to be consistent between checks. This makes it inconsistent by making it `self.agent_check.gauge` and so on.